samples/bluetooth/observer/boards/nrf52_bsim.conf

```diff
@@ -0,0 +1,7 @@
+# Example Zephyr Controller setting to receive Extended Advertising Report
+
+# Set maximum scan data length for Extended Scanning
+# CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=1650
```

Comment:
Is it needed to add a `nrf52_bsim.conf` when these configurations are already set in the `prj.conf`?